sphinx/config.py

```diff
@@ -209,6 +209,7 @@ class Config(object):
                            None),
         latex_logo = (None, None, string_classes),
         latex_appendices = ([], None),
+        latex_macros_onlywithprefix = (False, None),
```

Comment:
This represents the behavior correctly, but it is hard to understand what the option does from its name.
How about `latex_provides_old_macros` or `latex_use_old_macros`?

Anyway, please add a description of the option to `doc/config.rst`.